spec/save_operation_spec.cr

```diff
@@ -553,6 +572,20 @@ describe "Avram::SaveOperation" do
         end
       end
     end
+
+    context "when the default is false, but the field is required" do
+      it "passes required validation as 'false' is a valid Boolean value" do
```

Comment:
What do you think about this?

```suggestion
    context "when the default is false and the field is required" do
      it "is valid since 'false' is a valid Boolean value" do
```